examples/transfers/cancellations_test.go

```diff
@@ -0,0 +1,53 @@
+package transfers
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/moovfinancial/moov-go/pkg/moov"
+)
+
+func ExampleClient_CancelTransfer() {
+	mc, err := moov.NewClient()
+	if err != nil {
+		fmt.Errorf("new Moov client: %v", err)
+		return
+	}
+
+	var (
+		ctx              = context.Background()
+		partnerAccountID = "00000000-00000000-00000000-00000000"
+		transferID       = "00000000-00000000-00000000-00000000"
```

Comment:
maybe add a comment sayin to replace these with actual UUID values?

Reply:
it should be fine to leave is -- since it's an example, and should be assumed any IDs should be replaced w/ actual values from your app